commands/docker/docker-tag.md

```diff
@@ -0,0 +1,14 @@
+### Provide new tag to docker image
+
+`docker tag {{EXISTING_IMAGE_NAME[:TAG]/ID}} {{NEW_IMAGE_NAME[:TAG]}}}`
+
+- <b>EXISTING_IMAGE_NAME[:TAG]/ID: </b> Name with tag or image id of the image
```

Comment:
`EXISTING_IMAGE_NAME[:TAG]/ID` => `EXISTING_IMAGE_NAME[:TAG]`

Reply:
Updated in the new commit